src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/hooks/useProgramMetadata.js

```diff
@@ -48,12 +48,10 @@ export const useProgramMetadata = (programId: string) => {
         () => (optionSets ? optionSets.reduce(
             (acc, optionSet) => {
                 acc[optionSet.id] = {
-                    optionSet: {
```

Comment:
Are you sure this refactor is safe?

Reply:
If you have checked and it looks OK, then that is good enough for me :slightly_smiling_face:

Reply:
Thanks for raising it, but I couldn't find anything. Let me know if otherwise!